packages/core/src/client.ts

```diff
@@ -2175,7 +2175,9 @@ export class MedplumClient extends EventTarget {
     if (since) url.searchParams.set('_since', since);
 
     options.method = exportLevel ? 'GET' : 'POST';
-
+    options.headers = {};
+    options.headers['Accept'] = FHIR_CONTENT_TYPE;
```

Comment:
Should this be set for all client requests?

Reply:
As far as I know, it's needed for $export operations. For subsequent client requests such as for jobs/:jobid status page, it's not required.

Reply:
It is explicitly optional: https://www.hl7.org/fhir/http.html

But with that said, I like the idea of setting it by default - especially if we see other servers that incorrectly require it. 

It would be natural to add it to `addFetchOptionsDefaults`.